tests/AppBundle/Service/RestHandler/Report/DeputyCostsEstimateReportUpdateHandlerTest.php

```diff
@@ -225,4 +233,12 @@ private function assertNewProfDeputyEstimateCostIsCreated()
         $this->assertEquals(false, $profDeputyEstimateCost->getHasMoreDetails());
         $this->assertEquals(null, $profDeputyEstimateCost->getMoreDetails());
     }
+
+    public function costEstimateDataProvider()
```

Comment:
Handy little dataProvider. Can it live underneath the test to which it applies please?